src/main/java/org/owasp/wrongsecrets/challenges/cloud/Challenge11.java

```diff
@@ -139,4 +176,12 @@ private String getGCPChallenge11Value() {
         }
         return gcpDefaultValue;
     }
+
+    private String getAzureChallenge11Value() {
+        if (isAzure()) {
+            log.info(String.format("Using Azure Key Vault URI: %s", azureVaultUri));
```

Comment:
if it is the default value, maybe log that it is being skipped as it does not work then?

Reply:
Addressed!